packages/ckb-sdk-wallet/src/account.ts

```diff
@@ -2,8 +2,10 @@ import ECPair, { Options } from '@nervosnetwork/ckb-sdk-utils/lib/ecpair'
 import RPC from '@nervosnetwork/ckb-sdk-rpc'
 import { hexToBytes, lockScriptToHash } from '@nervosnetwork/ckb-sdk-utils'
 
+const CKBytes: CKBComponents.CapacityUnit.CKBytes = 10 ** 8
```

Comment:
Shouldn't this be `Shannon` instead as `1 CK Byte = 10 ** 8` Shannons?

Reply:
I've checked the [Ruby SDK](https://github.com/nervosnetwork/ckb-sdk-ruby/pull/34/files#diff-1d730354be973dfe6efbd3b77e084a9aR8] which sets `MIN_CELL_CAPACITY` to `40 * (10 ** 8)`

Reply:
I've checked the [Ruby SDK](https://github.com/nervosnetwork/ckb-sdk-ruby/pull/34/files#diff-1d730354be973dfe6efbd3b77e084a9aR8) which sets `MIN_CELL_CAPACITY` to `40 * (10 ** 8)`.

Reply:
Oh what I meant is, the variable (constant) is called `CKBytes`, which doesn't describe itself well and would cause confusion (at least for me it does cause that).

Reply:
I've renamed them in https://github.com/nervosnetwork/ckb-sdk-js/pull/118/commits/541d00afb9c7a9cf9f5328670f573dda4fdee8e3, is that more clear?